### DISCUSSION TITLE
Slash the Updater if they do not produce an update after a certain period of time

### DISCUSSION THREAD

**Discussion by anna-carroll** [celo-org/optics-monorepo#608] [Ideas]
On the Home.sol contract
- add a function that can be called by anyone that will slash the Updater's stake if a certain amount of time has passed since the a message has been added & an update has not occurred (timestamp should be timestamp of oldest message that has not also been updated)

Update documentation 

Other updates?

**Comment by prestwich** [comment]
thinking about this more, I kinda want to push it off to the sortition and governance design that comes later. maybe fold this into #66?

**Comment by prestwich** [comment]
gonna take it off this sprint for now 🤔

**Comment by anna-carroll** [comment]
@prestwich I think this definitely relates to the question about roles within the system, though I think the process of slashing the Updater for this doesn't have to be a governance task (because it can be easily proved on-chain by anyone and doesn't require any "trust")
